notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -49,9 +49,7 @@
   {
```

Comment:
<p>test comment 2</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
@amit2rrr see this from GH